src/flb_log_event_decoder.c

```diff
@@ -300,3 +300,51 @@ int flb_log_event_decoder_next(struct flb_log_event_decoder *context,
                                            event,
                                            &context->unpacked_event.data);
 }
+
+const char* flb_log_event_encoder_strerror(int result)
+{
+    switch (result) {
+    case FLB_EVENT_DECODER_SUCCESS:
```

Comment:
Indent the case statements one more level.

Reply:
Set a result variable on each case and return that variable at the end of the function.

Reply:
> Set a result variable on each case and return that variable at the end of the function.

I modified it.

Reply:
> Indent the case statements one more level.

It is the Apache C style guidelines.
https://httpd.apache.org/dev/styleguide.html#the-guidelines
```
Inside a switch() statement, the case keywords are indented to the same level as the switch line.
```

We use the style.
https://github.com/fluent/fluent-bit/blob/master/CONTRIBUTING.md#coding-style

Reply:
Sorry, my bad, I didn't realize and was in fact breaking the coding style myself...